CMakeLists.txt

```diff
@@ -14,6 +14,8 @@ set( CMAKE_CXX_EXTENSIONS FALSE CACHE BOOL "Disable (host) C++ extensions" )
 set( CMAKE_CUDA_STANDARD 17 CACHE STRING "The (CUDA) C++ standard to use" )
 set( CMAKE_CUDA_EXTENSIONS FALSE CACHE BOOL "Disable (CUDA) C++ extensions" )
 set( CMAKE_SYCL_STANDARD 17 CACHE STRING "The (SYCL) C++ standard to use" )
+set( CMAKE_HIP_STANDARD 17 CACHE STRING "The (HIP) C++ standard to use" )
+set( CMAKE_HIP_EXTENSIONS FALSE CACHE BOOL "Disable (HIP) C++ extensions" )
```

Comment:
Since we're using VecMem's HIP support here (Unless I'm very mistaken. Unfortunately the setup is not super clear on this, as I chose the same API for VecMem's HIP support that CMake implemented eventually.), this variable doesn't really do anything. That's why I also didn't add a `CMAKE_SYCL_EXTENSIONS` flag. As that wouldn't do anything either.

The variable is not harmful, but it may be well be removed...